pkg/kubectl/cmd/edit.go

```diff
@@ -93,7 +93,7 @@ func NewCmdEdit(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 		Example: fmt.Sprintf(editExample),
 		Run: func(cmd *cobra.Command, args []string) {
 			options.ChangeCause = f.Command(cmd, false)
-			if err := options.Complete(f, out, errOut, args); err != nil {
+			if err := options.Complete(f, out, errOut, args, cmd); err != nil {
```

Comment:
@mengqiy No. This is the side effect since `options.Complete` has been changed. We've not planed adding `include-uninitialized` flag to `kubectl edit`. @caesarxuchao @ahmetb Correct me if I'm wrong.

I've added below codes to method `Complete(...)`, so that it will not break current `kubectl edit`.

```Go
if cmd.Flags().Lookup("include-uninitialized") != nil {
        includeUninitialized = cmdutil.GetFlagBool(cmd, "include-uninitialized")
}
```